dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/BrowserAPI.java

```diff
@@ -190,21 +204,13 @@ public WfData(Contentlet contentlet, List<Integer> permissions, User user, boole
 
 		List<Map<String, Object>> returnList = new ArrayList<Map<String, Object>>();
 
-		Role[] roles = new Role[] {};
-		try {
-			roles = com.dotmarketing.business.APILocator.getRoleAPI()
+		Role[] roles = com.dotmarketing.business.APILocator.getRoleAPI()
 					.loadRolesForUser(user.getUserId()).toArray(new Role[0]);
-		} catch (DotDataException e1) {
-			Logger.error(BrowserAPI.class, e1.getMessage(), e1);
-		}
+
 
 		// gets folder parent
-		Folder parent = null;
-		try {
-			parent = APILocator.getFolderAPI().find(folderId, user, false);
-		} catch (Exception ignored) {
-			// Probably what we have here is a host
-		}
+		Folder parent = APILocator.getFolderAPI().find(folderId, user, false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'parent' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)